styles/style.css

```diff
@@ -1104,45 +810,29 @@ h1 a {
 
 .body li:nth-child(4n+0) {
   -webkit-animation: glitch-in 0.2s linear 0.5s 3;
-  -moz-animation: glitch-in 0.2s linear 0.5s 3;
-  -ms-animation: glitch-in 0.2s linear 0.5s 3;
   animation: glitch-in 0.2s linear 0.5s 3;
   -webkit-animation-fill-mode: both;
-  -moz-animation-fill-mode: both;
-  -ms-animation-fill-mode: both;
   animation-fill-mode: both;
 }
 
 .body li:nth-child(4n+3) {
   -webkit-animation: glitch-in 0.3s linear 0.7s 2;
-  -moz-animation: glitch-in 0.3s linear 0.7s 2;
-  -ms-animation: glitch-in 0.3s linear 0.7s 2;
   animation: glitch-in 0.3s linear 0.7s 2;
   -webkit-animation-fill-mode: both;
-  -moz-animation-fill-mode: both;
-  -ms-animation-fill-mode: both;
   animation-fill-mode: both;
 }
 
 .body li:nth-child(4n+1) {
   -webkit-animation: glitch-in 0.4s linear 0.6s 1;
-  -moz-animation: glitch-in 0.4s linear 0.6s 1;
-  -ms-animation: glitch-in 0.4s linear 0.6s 1;
   animation: glitch-in 0.4s linear 0.6s 1;
   -webkit-animation-fill-mode: both;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The property -moz-animation is compatible with -webkit-animation and should be included as well.](https://www.codacy.com/app/j-hasseriis/cepheus/file/1888804921/issues/source?bid=3085146&fileBranchId=3088807#l833)